api/src/main/java/io/grpc/EquivalentAddressGroup.java

```diff
@@ -35,6 +35,13 @@
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1770")
 public final class EquivalentAddressGroup {
 
+  /**
+   * The authority to be used when constructing Subchannels for this EquivalentAddressGroup.
+   */
+  @Attr
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/6138")
+  public static final Attributes.Key<String> ATTR_AUTHORITY_OVERRIDE =
+      Attributes.Key.create("io.grpc.grpclb.authorityOverride");
```

Comment:
I think this is not related to `io.grpc.grpclb`. Maybe `io.grpc.EquivalentAddressGroup.authorityOverride` is a better name.

Reply:
Done. I went ahead and made this change, since it was trivial.